src/Symfony/Component/HttpClient/Response/AsyncResponse.php

```diff
@@ -117,11 +117,20 @@ public function getHeaders(bool $throw = true): array
 
     public function getInfo(?string $type = null)
     {
+        if ('debug' === ($type ?? 'debug')) {
+            $debug = implode('', array_column($this->info['previous_info'] ?? [], 'debug'));
```

Comment:
the debug info for AsyncResponse should contain all debug statements from previous requests, otherwise it's difficult to debug (happened to me)